crates/agglayer-storage/src/columns/proof_per_certificate/mod.rs

```diff
@@ -0,0 +1,30 @@
+use bincode::Options;
+
+use super::{default_bincode_options, Codec, ColumnSchema, PROOF_PER_CERTIFICATE_CF};
+use crate::types::{CertificateId, Proof};
+
+#[cfg(test)]
+mod tests;
+
+/// Column family that returns the generated proof for one certificate.
+///
+/// | --- key ------  |    | --- value ---- |
+/// | CertificateId   | => | Proof bytes    |
+pub struct ProofPerCertificateColumn;
+
+impl Codec for Vec<u8> {
+    fn encode(&self) -> Result<Vec<u8>, crate::error::Error> {
+        Ok(default_bincode_options().serialize(&self)?)
```

Comment:
Nit: this pattern involving `Ok(default_bincode_options().serialize(&self)?)` could be a helper function in the module where `Codec` is defined.

Reply:
Indeed, I will put a default trait function